### ISSUE TITLE
completeExample doesn't compile

### ISSUE THREAD

**Issue by Greece2** [s00500/ESPUI#245]
When trying to compile the example "completeExample", I get the error message:
```
C:\Users\XYZ\AppData\Local\Temp\ccyksZ49.s: Assembler messages:
C:\Users\XYZ\AppData\Local\Temp\ccyksZ49.s:6410: Internal error in expression_maybe_register at /workdir/repo/binutils-gdb-gnu/gas/config/tc-xtensa.c:1938.
Please report this bug.
```
No matter if IDE 2.1.1 or IDE 1.8.19 is used.
Board is "Generic ESP8266 module", same problem with "NodeMCU1.0" and other ESP8266 variants.

**Comment by Greece2** [comment]
Yes, all other ESPUI examples and other projects compile fine. No, don't think I'll try PIO, too much stuff on the PC already ;-)

**Comment by iangray001** [comment]
Well it is still a compiler bug. If you are feeling public-spirited you could report it, but I would assume that the Arduino IDE is probably using an old compiler anyway. I've not seen that error before.
